flair/datasets/entity_linking.py

```diff
@@ -1760,3 +2210,398 @@ def __init__(
+            if result_path.exists():
+                continue
+
+            path = cached_path(url, data_folder)
+            unpack_file(path, data_folder)
+
+        return results_files
+
+    def _file_to_dicts(self, filepath: Path) -> Iterator[Dict[str, Any]]:
+        with open(filepath) as f:
+            for doc in pubtator.iterparse(f):
+                unified_example = {
+                    "id": doc.pmid,
+                    "document_id": doc.pmid,
+                    "passages": [
+                        {
+                            "text": [doc.title],
+                            "offsets": [[0, len(doc.title)]],
+                        },
+                        {
+                            "text": [doc.abstract],
+                            "offsets": [
+                                [
+                                    # +1 assumes the title and abstract will be joined by a space.
+                                    len(doc.title) + 1,
+                                    len(doc.title) + 1 + len(doc.abstract),
+                                ]
+                            ],
+                        },
+                    ],
+                }
+
+                unified_entities = []
+                for i, entity in enumerate(doc.annotations):
+                    # We need a unique identifier for this entity, so build it from the document id and entity id
+                    unified_entity_id = "_".join([doc.pmid, entity.id, str(i)])
+                    # The user can provide a callable that returns the database name.
+                    normalized = []
+
+                    for x in entity.id.split("|"):
+                        if x.startswith(("OMIM", "omim")):
+                            normalized.append({"db_name": "OMIM", "db_id": x.strip().split(":")[-1]})
+                        elif "+" in x:
+                            normalized.extend(
+                                [
+                                    {
+                                        "db_name": "MESH",
+                                        "db_id": y.split(":")[-1].strip(),
+                                    }
+                                    for y in x.split("+")
+                                ]
+                            )
+                        else:
+                            normalized.append({"db_name": "MESH", "db_id": x.split(":")[-1].strip()})
+
+                    unified_entities.append(
+                        {
+                            "id": unified_entity_id,
+                            "type": entity.type,
+                            "text": [entity.text],
+                            "offsets": [[entity.start, entity.end]],
+                            "normalized": normalized,
+                        }
+                    )
+
+                unified_example["entities"] = unified_entities
+
+                yield unified_example
+
+
+class BIGBIO_BC5CDR_CHEMICAL(BigbioCorpus):
```

Comment:
Also for each of the BigbioCorpus subclasses please add a brief description of the dataset and link.